x-pack/plugins/canvas/canvas_plugin_src/renderers/markdown/index.js

```diff
@@ -22,7 +21,7 @@ export const markdown = () => ({
 
     /* eslint-disable react/no-danger */
     ReactDOM.render(
-      <div className="canvasMarkdown" style={fontStyle} dangerouslySetInnerHTML={html} />,
+      <div className="kbnMarkdown__body" style={fontStyle} dangerouslySetInnerHTML={html} />,
```

Comment:
Can you keep the `canvasMarkdown` class on here as well? All the other renderers follow that pattern, and it gives us a reliable handle that's not bound to a style class (which may change).

Reply:
Yep. In general when seeing this stuff I wondered why it wasn't a data attribute for the same reason. But I'll add it back in.